### PR TITLE
zh-translation: docs/setup/install/virtual machine/index.md

### PR DESCRIPTION
**Pull request by MXuDong**
add file: experimental.md

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/setup/install/virtual-machine/index.md [hunk 1]
```diff
@@ -75,109 +113,212 @@ test: no
...
 
-1. 使用 `istioctl x workload entry` 命令生成：
-       * `cluster.env`: 包含用来识别名称空间、服务帐户、网络 CIDR、和入站端口(可选)的元数据。
-       * `istio-token`: 用来从 CA 获取证书的 Kubernetes 令牌。
-       * `mesh.yaml`: 提供网络名称、信任域、和一些其他值的附加 Istio 元数据。
-       * `root-cert.pem`: 用于认证的根证书。
-       * `hosts`:  `/etc/hosts` 的补充，代理将使用该补充从 Istiod 获取 xDS.*。
+{{< warning >}}
+在开始生成 `istio-token` 之前，作为 `istioctl x workload entry` 的一部分，您应该按照[文档](/zh/docs/ops/best-practices/security/#configure-third-party-service-account-tokens)来验证集群中是否开启了第三方 `token` 。
+如果没有开启第三方 token，您应该为 Istio 安装指令添加参数 `--set values.global.jwtPolicy=first-party-jwt` 。
```
Comment (kebe7jun) [inline content/zh/docs/setup/install/virtual-machine/index.md:279]: 第三方 token 在 Kubernetes 官方叫 `服务帐户令牌`，你看看可以怎么组织语言。

https://kubernetes.io/zh/docs/tasks/configure-pod-container/configure-service-account/

Comment (MXuDong) [reply content/zh/docs/setup/install/virtual-machine/index.md]: 已经修改

**Review by kebe7jun — APPROVED** [review]

**Review by rootsongjc — APPROVED** [review]
/LGTM

**Pull request merged**